Translate Getting Started (#60)
reactjs/id.reactjs.org: content/docs/getting-started.md

  - "docs/environments.html"
---

> Laman ini adalah sebuah ikhtisar dari dokumentasi React dan sumber - sumber terkait.

Reviewer: ```suggestion
Laman ini adalah sebuah ikhtisar dari dokumentasi React dan sumber-sumber terkait.
```

> **React** adalah sebuah *JavaScript library* untuk membangun tampilan user (*user interfaces*). Pelajari apa sebenarnya React itu di [Laman Beranda kami](/) atau [di dalam tutorial](/tutorial/tutorial.html).

Reviewer: ```suggestion
**React** adalah sebuah *library* JavaScript untuk membangun antarmuka pengguna (*user interface*). Pelajari apa sebenarnya React itu di [beranda kami](/) atau [di dalam tutorial](/tutorial/tutorial.html).
```

---

> - [Coba React](#coba-react)

Reviewer 2: I don't think we should translate the hyperlink. All other languages translation left it as it is. This should still be `#try-react`. Reference https://es.reactjs.org/docs/getting-started.html#try-react

- [Belajar React](#belajar-react)
- [Tetap Terinformasi](#tetap-terinformasi)
- [Dokumentasi Berversi](#dokumentasi-berversi)

> - [Sesuatu Hilang?](#sesuatu-hilang)

Reviewer: `Ada yang Hilang?` might be more relevant.

> ## Coba React {#coba-react}

Reviewer 2: See my previous comment

> React telah dirancang dari awal untuk adopsi bertahap, dan **anda dapat menggunakan sedikit atau sebanyak yang diperlukan.** Baik anda ingin mencicipi menggunakan React, menambahkan beberapa interaktif ke sebuah laman *HTML* sederhana, atau memulai sebuah aplikasi React yang kompleks, tautan di bagian ini akan membantu anda untuk memulai.

Reviewer: ```suggestion
React telah dirancang dari awal untuk adopsi bertahap, dan **anda dapat menggunakan sedikit atau sebanyak yang diperlukan.** Baik anda ingin mencicipi menggunakan React, menambahkan beberapa interaktivitas ke sebuah laman HTML sederhana, atau memulai sebuah aplikasi React yang kompleks, tautan di bagian ini akan membantu anda untuk memulai.
```

### Online Playgrounds {#online-playgrounds}

> Jika anda tertarik bermain dengan React, anda dapat menggunakan sebuah *online code playground*. Coba sebuah *Hello World template* di [CodePen](codepen://hello-world) atau [CodeSandbox](https://codesandbox.io/s/new).

Reviewer: ```suggestion
Jika anda tertarik bermain dengan React, Anda dapat menggunakan sebuah *online code playground*. Coba sebuah *template* Hello World di [CodePen](codepen://hello-world) atau [CodeSandbox](https://codesandbox.io/s/new).
```

> Jika anda tertarik bermain dengan React, Anda dapat menggunakan sebuah *online code playground*. Coba sebuah *template* Hello World di [CodePen](codepen://hello-world) atau [CodeSandbox](https://codesandbox.io/s/new).

Reviewer 2: I don't think you need both `sebuah` in here. Kinda weird to read IMO

> Jika anda memilih untuk menggunakan *text editor* sendiri, anda juga dapat [Unduh berkas HTML ini](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit, dan buka itu dari *local filesystem* pada peramban anda. Itu melakukan perubahan *runtime code* yang lambat, jadi kami merekomendasikan menggunakan ini untuk demo sederhana.

Reviewer: ```suggestion
Jika anda memilih untuk menggunakan *text editor* sendiri, Anda juga dapat [mengunduh berkas HTML ini](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), sunting, dan buka itu dari *filesystem* lokal pada peramban Anda. Hal tersebut memperlambat transformasi *runtime code*, jadi kami merekomendasikan menggunakan ini untuk demonstrasi sederhana.
```

### Tambahkan React ke sebuah Website {#add-react-to-a-website}

> Anda dapat [menambahkan React ke sebuah laman HTML dalam satu menit](/docs/add-react-to-a-website.html). Anda juga dapat secara bertahap memperluas kehadirannya, atau menyimpannya di beberapa *widget dynamic*.

Reviewer: ```suggestion
Anda dapat [menambahkan React ke sebuah laman HTML dalam satu menit](/docs/add-react-to-a-website.html). Anda juga dapat secara bertahap memperluas kehadirannya, atau menyimpannya di beberapa *widget* dinamis.
```

### Buat Aplikasi React Baru {#create-a-new-react-app}

> Ketika memulai proyek React,  laman HTML sederhana dengan *script tags*](/docs/add-react-to-a-website.html) mungkin masih menjadi pilihan terbaik. Hanya membutuhkan waktu semenit untuk memulai!

Reviewer 3: ```suggestion
Ketika memulai proyek React, laman HTML sederhana dengan *script tags*](/docs/add-react-to-a-website.html) mungkin masih menjadi pilihan terbaik. Hanya membutuhkan waktu semenit untuk memulai!
```

Remove double space.

> Ketika memulai sebuah proyek React, [sebuah laman HTML sederhana dengan *script tags*](/docs/add-react-to-a-website.html) mungkin masih menjadi pilihan terbaik. Itu hanya membutuhkan waktu semenit untuk memulai!

Reviewer 2: Rather than `Itu hanya membutuhkan waktu semenit untuk memulai!`, i think removing `itu` would result in better readability so it becomes `Hanya membutuhkan waktu semenit untuk memulai!`

> Saat aplikasi Anda bertumbuh, Anda mungkin ingin mempertimbangkan pengaturan yang lebih terintegrasi. Ada [beberapa JavaScript *toolchain*](/docs/create-a-new-react-app.html) yang kami rekomendasikan untuk aplikasi besar. Masing-masing dari mereka dapat bekerja dengan sedikit atau tanpa konfigurasi dan memungkinkan Anda mendapatkan keuntungan penuh dari ekosistem React.

Reviewer 3: Comparing `bertumbuh` with `berkembang` and `ingin` dengan `perlu`, which one is better for this context?

```suggestion
Saat aplikasi Anda berkembang, Anda mungkin perlu mempertimbangkan pengaturan yang lebih terintegrasi. Ada [beberapa JavaScript *toolchain*](/docs/create-a-new-react-app.html) yang kami rekomendasikan untuk aplikasi besar. Masing-masing dari mereka dapat bekerja dengan sedikit atau tanpa konfigurasi dan memungkinkan Anda mendapatkan keuntungan penuh dari ekosistem React.
```

> Saat aplikasi anda tumbuh, Anda mungkin ingin mempertimbangkan pengaturan yang lebih terintegrasi. Ada [beberapa *JavaScript toolchains*](/docs/create-a-new-react-app.html) kami merekomendasikan untuk aplikasi besar. Setiap dari mereka dapat bekerja dengan sedikit atau tanpa konfigurasi dan memungkinkan anda mendapatkan keuntungan penuh dari ekosistem React.

Reviewer: ```suggestion
Saat aplikasi Anda bertumbuh, Anda mungkin ingin mempertimbangkan pengaturan yang lebih terintegrasi. Ada [beberapa JavaScript *toolchain*](/docs/create-a-new-react-app.html) yang kami rekomendasikan untuk aplikasi besar. Masing-masing dari mereka dapat bekerja dengan sedikit atau tanpa konfigurasi dan memungkinkan Anda mendapatkan keuntungan penuh dari ekosistem React.
```

## Belajar React {#belajar-react}

> Orang orang data ke React dari latar belakang yang berbeda dan dengan cara belajar yang berbeda. Apakah anda lebih suka pendekatan yang lebih teoritis atau praktis, kami harap anda akan menemukan bagian yang bermanfaat.

Reviewer: ```suggestion
Orang-orang data ke React dari latar belakang yang berbeda dan dengan cara belajar yang berbeda-beda. Apakah Anda lebih suka pendekatan yang lebih teoritis atau praktis, kami harap anda akan menemukan bagian yang bermanfaat.
```

> Orang-orang data ke React dari latar belakang yang berbeda dan dengan cara belajar yang berbeda-beda. Apakah Anda lebih suka pendekatan yang lebih teoritis atau praktis, kami harap anda akan menemukan bagian yang bermanfaat.

Reviewer 2: Suggestion:
`Orang datang menggunakan React dari berbagai latar belakang dan dengan cara belajar yang berbeda-beda.`

> * Jika anda lebih suka **belajar sambil melakukan**, mulai dengan [tutorial praktis](/tutorial/tutorial.html) kami.

Reviewer: ```suggestion
* Jika anda lebih suka **belajar sambil melakukan**, mulailah dengan [tutorial praktis](/tutorial/tutorial.html) kami.
```

> * Jika anda lebih suka **belajar sambil melakukan**, mulailah dengan [tutorial praktis](/tutorial/tutorial.html) kami.

Reviewer 2: `learn by doing` should have consistent translation. In here, you translate it as `belajar sambil melakukan` while below, you translate it as `belajar dengan mempraktikkan`

> * Jika anda lebih suka **belajar konsep langkah demi langkah**, mulai dengan [panduan untuk konsep utama](/docs/hello-world.html) kami.

Reviewer: ```suggestion
* Jika anda lebih suka **belajar konsep langkah demi langkah**, mulailah dengan [panduan untuk konsep utama](/docs/hello-world.html) kami.
```

> Seperti halnya teknologi yang tidak dikenal, React memiliki kurva belajar. Dengan latihan dan kesabaran, Anda akan bertahan.

Reviewer 2: `learning curve` should be translated as `kurva pembelajaran` i think.

Other suggestion: `Dengan latihan dan kesabaran, Anda *akan* bisa menguasainya.`

> Seperti halnya teknologi yang tidak dikenal, React memiliki kurva belajar. Dengan latihan dan beberapa kesabaran, anda akan bertahan. With practice and some patience, you *will* get the hang of it.

Reviewer: ```suggestion
Seperti halnya teknologi yang tidak dikenal, React memiliki kurva belajar. Dengan latihan dan kesabaran, Anda akan bertahan.
```

### Contoh Pertama {#first-examples}

> [Laman Beranda React](/) berisi beberapa contoh kecil React dengan sebuah *live editor*. contains a few small React examples with a live editor. Even if you don't know anything about React yet, try changing their code and see how it affects the result.

Reviewer: ```suggestion
[Beranda React](/) berisi beberapa contoh kecil React dengan sebuah *live editor*. Bahkan jika Anda tidak mengetahui apapun tentang React sebelumnya, cobalah untuk mengubah kodenya dan lihat hasilnya secara langsung.
```

### React untuk Pemula {#react-for-beginners}

> Jika anda merasa dokumentasi React terlalu sulit untuk anda ikuti, cek [ikhtisar React ini oleh Tania Rascia](https://www.taniarascia.com/getting-started-with-react/). Ini memperkenalkan konsep terpenting React secara terperinci, mudah untuk pemula. Setelah selesai, coba dokumentasi yang lain! 

Reviewer: ```suggestion
Jika Anda merasa dokumentasi React terlalu sulit untuk Anda ikuti, cek [ikhtisar React ini oleh Tania Rascia](https://www.taniarascia.com/getting-started-with-react/). Ini memperkenalkan konsep terpenting React secara terperinci, mudah untuk pemula. Setelah selesai, cobalah dokumentasi yang lain! 
```

### React untuk Desainer {#react-for-designers}

> Jika Anda datang dari latar belakang desain, [sumber-sumber ini](http://reactfordesigners.com/) adalah tempat yang bagus untuk memulai.

Reviewer 4: ```suggestion
Jika Anda datang dari latar belakang desain, [sumber-sumber ini](https://reactfordesigners.com/) adalah tempat yang bagus untuk memulai.
```

Laman `https` jangan diubah ke `http` ya.

> Jika anda datang dari latar belakang desain, [Sumber daya ini](http://reactfordesigners.com/) adalah tempat yang bagus untuk memulai.

Reviewer: ```suggestion
Jika Anda datang dari latar belakang desain, [sumber-sumber ini](http://reactfordesigners.com/) adalah tempat yang bagus untuk memulai.
```

### Sumber Daya JavaScript {#javascript-resources}

> Dokumentasi React mengasumsikan anda sudah terbiasa dengan *programming* menggunakan bahasa Javascript. Anda tidak harus ahli, tetapi ini akan lebih susah untuk belajar React dan Javascript di waktu yang sama.

Reviewer: ```suggestion
Dokumentasi React mengasumsikan Anda sudah terbiasa dengan pemrograman menggunakan bahasa JavaScript. Anda tidak harus ahli, tetapi ini akan lebih sulit untuk belajar React dan JavaScript di waktu yang bersamaan.
```

> Kami sarankan gunakan [ikhtisar Javascript ini](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) untuk cek kemampuan pengetahuan anda. Ini akan membutuhkan waktu sekitar 30 menit sampai dengan satu jam tetapi anda akan merasa percaya diri untuk belajar React.

Reviewer: ```suggestion
Kami sarankan untuk menggunakan [ikhtisar JavaScript ini](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) untuk memeriksa tingkatan pengetahuan Anda. Ini akan membutuhkan waktu sekitar 30 menit sampai dengan satu jam tetapi Anda akan merasa lebih percaya diri untuk belajar React.
```

>Tip
>

> >Ketika anda bingung dengan sesuatu pada Javascript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) dan [javascript.info](http://javascript.info/) adalah website yang bagus untuk anda cek. Ada juga [forum bantuan komunitas](/community/support.html) dimana anda dapat meminta bantuan.

Reviewer: ```suggestion
>Ketika Anda dibingungkan oleh sesuatu pada JavaScript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) dan [javascript.info](http://javascript.info/) adalah _website_ yang bagus untuk Anda periksa. Ada juga [forum bantuan komunitas](/community/support.html) dimana Anda dapat meminta bantuan.
```

### Tutorial Praktis {#practical-tutorial}

> Jika Anda lebih suka **belajar dengan mempraktikkan** periksa [tutorial](/tutorial/tutorial.html) kami. Dalam tutorial ini, kami membuat game tic-tac-toe dengan React. Anda mungkin tergoda untuk melewati karena Anda tidak membuat game -- tetapi berikanlah kesempatan untuk mencoba. Teknik yang akan Anda pelajari dalam tutorial ini sangat penting untuk membangun aplikasi *apapun* dengan React, dan menguasainya akan memberi Anda pemahaman yang lebih dalam.

Reviewer 2: `learn by doing` should have consistent translation. In here, you translate it as `belajar dengan mempraktikkan` while on my previous comment, you translate it as `belajar sambil melakukan`

> Jika anda lebih suka **belajar dengan mempraktikkan** cek [tutorial](/tutorial/tutorial.html) kami. Dalam tutorial ini, kami membuat game tic-tac-toe game dengan React. Anda mungkin tergoda untuk melewati karena Anda tidak membuat game -- tetapi beri kesempatan. Teknik yang anda pelajari dalam tutorial ini sangat penting untuk membangun aplikasi *apa saja* dengan React, dan menguasainya akan memberi Anda pemahaman yang lebih dalam.

Reviewer: ```suggestion
Jika Anda lebih suka **belajar dengan mempraktikkan** periksa [tutorial](/tutorial/tutorial.html) kami. Dalam tutorial ini, kami membuat game tic-tac-toe dengan React. Anda mungkin tergoda untuk melewati karena Anda tidak membuat game -- tetapi berikanlah kesempatan untuk mencoba. Teknik yang akan Anda pelajari dalam tutorial ini sangat penting untuk membangun aplikasi *apapun* dengan React, dan menguasainya akan memberi Anda pemahaman yang lebih dalam.
```

> ### Panduan langkah demi langkah {#step-by-step-guide}

Reviewer: ```suggestion
### Panduan Langkah demi Langkah {#step-by-step-guide}
```

> Jika anda lebih suka untuk **belajar konsep langkah demi langkah**, [panduan konsep utama](/docs/hello-world.html) adalah tempat terbaik untuk memulai. Setiap bab berikutnya di dalamnya terbuat dari pengetahuan yang diperkenalkan pada bab sebelumnya sehingga anda tidak akan kehilangan apapun saat anda melanjutkan.

Reviewer: ```suggestion
Jika Anda lebih suka untuk **belajar konsep langkah demi langkah**, [panduan konsep utama](/docs/hello-world.html) adalah tempat terbaik untuk memulai. Setiap bab berikutnya di dalamnya terbuat dari pengetahuan yang diperkenalkan pada bab sebelumnya sehingga Anda tidak akan kehilangan apapun saat Anda melanjutkan belajarnya.
```

### Berpikir dalam React {#thinking-in-react}

> Banyak user-user React yang membaca kredit [Thinking in React](/docs/thinking-in-react.html) sebagai momen dimana React akhirnya "clicked" untuk mereka. Ini mungkin panduan React tertua tetapi masih relevan.

Reviewer 2: Suggestion: `Banyak pengguna React mengakui membaca [Thinking in React](/docs/thinking-in-react.html) sebagai momen dimana React akhirnya "clicked" untuk mereka. Ini mungkin panduan React yang paling tua tetapi masih relevan.`

> Banyak user - user React Many React users credit reading [Thinking in React](/docs/thinking-in-react.html) as the moment React finally "clicked" for them. It's probably the oldest React walkthrough but it's still just as relevant.

Reviewer: This is unfinished

### Rekomendasi Kursus {#recommended-courses}

> Terkadang orang mencari buku dan video kursus lebih membantu daripada dokumentasi resmi. Kami memelihara [daftar sumber daya yang umum di rekomendasikan](/community/courses.html), beberapa diantaranya gratis.

Reviewer: ```suggestion
Terkadang orang mencari buku dan video kursus lebih membantu daripada dokumentasi resmi. Kami memelihara [daftar sumber daya rekomendasi yang umum](/community/courses.html), beberapa di antaranya gratis.
```

### Konsep Lanjutan {#advanced-concepts}

> Setelah Anda merasa nyaman dengan [Konsep Utama](#main-concepts) dan bermain dengan *React*, Anda mungkin tertarik dengan topik yang lebih lanjut. Bagian ini akan memperkenalkan Anda pada fitur React yang *powerful*, tetapi kurang umum digunakan seperti [context](/docs/context.html) and [refs](/docs/refs-and-the-dom.html).

Reviewer: ```suggestion
Setelah Anda merasa nyaman dengan [konsep utama](#main-concepts) dan bermain dengan React sedikit, Anda mungkin tertarik dengan topik yang lebih lanjut. Bagian ini akan memperkenalkan Anda pada fitur React yang *powerful*, tetapi kurang umum digunakan seperti [context](/docs/context.html) dan [refs](/docs/refs-and-the-dom.html).
```

### Referensi API {#api-reference}

> Bagian dokumentasi ini berguna ketika anda ingin belajar lebih detail mengenai *React API*.  Bagian dokumentasi ini berguna ketika Anda ingin belajar lebih jauh mengenai React API. Misalnya, [referensi API `React.Component`](/docs/react-component.html) dapat memberi anda detail dari bagaimana `setState()` bekerja, dan perbedaan serta kegunaan metode *lifecycle*.

Reviewer 3: Remove double space.

```suggestion
Bagian dokumentasi ini berguna ketika anda ingin belajar lebih detail mengenai *React API*. Bagian dokumentasi ini berguna ketika Anda ingin belajar lebih jauh mengenai React API. Misalnya, [referensi API `React.Component`](/docs/react-component.html) dapat memberi anda detail dari bagaimana `setState()` bekerja, dan perbedaan serta kegunaan metode *lifecycle*.
```

> Bagian dokumentasi ini berguna ketika anda ingin belajar lebih detail mengenai *React API*.  This documentation section is useful when you want to learn more details about a particular React API. Misalnya, [`React.Component` API reference](/docs/react-component.html) dapat memberi anda detail dari bagaimana `setState()` bekerja, dan perbedaan serta kegunaan metode *lifecycle*.

Reviewer: This is unfinished.

### Glosarium dan FAQ {#glossary-and-faq}

> [Glosari](/docs/glossary.html) berisi ikhtisar paling umum yang akan ada lihat pada dokumentasi React. Ada juga bagian *FAQ* yang didedikasikan pertanyaan dan jawaban singkat tentang topik umum, termasuk [membuat AJAX requests](/docs/faq-ajax.html), [component state](/docs/faq-state.html), dan [file structure](/docs/faq-structure.html).

Reviewer 2: This `glossary` translation also inconsistent. On the heading you use `Glosarium` while in here you use `Glosari`. Both should be `Glosarium`

> [Glosari](/docs/glossary.html) berisi ikhtisar paling umum yang akan ada lihat pada dokumentasi *React*. Ada juga bagian *FAQ* yang didedikasikan pertanyaan dan jawaban singkat tentang topik umum, termasuk [membuat AJAX requests](/docs/faq-ajax.html), [component state](/docs/faq-state.html), dan [file structure](/docs/faq-structure.html).

Reviewer: ```suggestion
[Glosari](/docs/glossary.html) berisi ikhtisar paling umum yang akan Anda lihat pada dokumentasi *React*. Ada juga bagian FAQ yang didedikasikan pertanyaan dan jawaban singkatnya tentang topik umum, termasuk [membuat AJAX *request*](/docs/faq-ajax.html), [component state](/docs/faq-state.html), dan [file structure](/docs/faq-structure.html).
```

> ## Tetap terinformasi {#staying-informed}

Reviewer: ```suggestion
## Tetap Terinformasi {#staying-informed}
```

> [React blog](/blog/) adalah sumber resmi untuk pembaruan dari tim *React*. Apapun yang penting, termasuk catatan rilis atau pemberitahuan penghentian, akan di posting terlebih dahulu disana.

Reviewer: ```suggestion
[React blog](/blog/) adalah sumber resmi untuk pembaruan dari tim *React*. Apapun yang penting, termasuk catatan rilis atau pemberitahuan penghentian, akan di posting terlebih dahulu di sana.
```

> Anda juga dapat mengikuti [akun @reactjs](https://twitter.com/reactjs) di Twitter, tetapi anda tidak akan kehilangan sesuatu yang penting jika hanya membaca blog.

Reviewer: ```suggestion
Anda juga dapat mengikuti [akun @reactjs](https://twitter.com/reactjs) di Twitter, tetapi Anda tidak akan kehilangan sesuatu yang penting jika hanya membaca blog.
```

> Tidak setiap rilis React mendapatkan posting sendiri pada blog, tapi anda dapat menemukan *changelog* terperinci untuk setiap rilis [di file `CHANGELOG.md` pada repositori React](https://github.com/facebook/react/blob/master/CHANGELOG.md), dan juga pada laman [Rilis](https://github.com/facebook/react).

Reviewer: ```suggestion
Tidak setiap rilis React mendapatkan postingan sendiri pada blog, tapi Anda dapat menemukan *changelog* terperinci untuk setiap rilis [di berkas `CHANGELOG.md` pada repositori React](https://github.com/facebook/react/blob/master/CHANGELOG.md), dan juga pada laman [Rilis](https://github.com/facebook/react).
```

## Dokumentasi Berversi {#dokumentasi-berversi}

> Dokumentasi ini selali mencerminkan versi stabil terbaru dari React. Sejak React 16, Anda dapat menemukan versi dokumentasi yang lebih lama [pada halaman terpisah](/versions). Perhatikan dokumentasi untuk versi yang lama adalah *snapshotted* pada saat rilis, dan tidak diperbaharui.

Reviewer: ```suggestion
Dokumentasi ini selalu mencerminkan versi stabil terbaru dari React. Sejak React 16, Anda dapat menemukan versi dokumentasi yang lebih lama [pada halaman terpisah](/versions). Perhatikan dokumentasi untuk versi yang lama adalah *snapshotted* pada saat rilis, dan tidak diperbaharui.
```

> ## Sesuatu Hilang? {#sesuatu-hilang}

Reviewer: `Ada yang Hilang?` might be more relevant.

> Jika ada sesuatu yang hilang di dalam dokumentasi atau  jika anda menemukan bagian yang membingungkan, Mohon untuk [mengajukan issue pada repositori dokumentasi](https://github.com/reactjs/reactjs.org/issues/new) dengan saran Anda untuk peningkatan, atau tweet di [akun @reactjs](https://twitter.com/reactjs). Kami suka mendengarnya darimu!

Reviewer 3: Remove double space.

```suggestion
Jika ada sesuatu yang hilang di dalam dokumentasi atau jika anda menemukan bagian yang membingungkan, Mohon untuk [mengajukan issue pada repositori dokumentasi](https://github.com/reactjs/reactjs.org/issues/new) dengan saran Anda untuk peningkatan, atau tweet di [akun @reactjs](https://twitter.com/reactjs). Kami suka mendengarnya darimu!
```

> Jika ada sesuatu yang hilang di dalam dokumentasi atau  jika anda menemukan bagian yang membingungkan, Mohon untuk [mengajukan file issue pada dokumentasi repositori](https://github.com/reactjs/reactjs.org/issues/new) dengan saran Anda untuk peningkatan, atau tweet di akun [@reactjs account](https://twitter.com/reactjs). Kami suka mendengarnya darimu!

Reviewer 4: ```suggestion
Jika ada sesuatu yang hilang di dalam dokumentasi atau  jika anda menemukan bagian yang membingungkan, Mohon untuk [mengajukan issue pada repositori dokumentasi](https://github.com/reactjs/reactjs.org/issues/new) dengan saran Anda untuk peningkatan, atau tweet di [akun @reactjs](https://twitter.com/reactjs). Kami suka mendengarnya darimu!
```

Sedikit perubahan untuk terjemahan yang agak rancu. 🙏

> Jika ada sesuatu yang hilang di dalam dokumentasi atau  jika anda menemukan bagian yang membingungkan, Mohon [file an issue for the documentation repository](https://github.com/reactjs/reactjs.org/issues/new) with your suggestions for improvement, or tweet at the [@reactjs account](https://twitter.com/reactjs). Kami suka mendengarnya darimu!

Reviewer: This is unfinished.
